client-app/app/templates/index.hbs

```diff
@@ -112,6 +121,35 @@
           <LinkTo @route="settings" class="settings btn no-text">
             <FaIcon @icon="cog" />
           </LinkTo>
+
+          {{#if this.buildingGroupingPattern}}
+            <button
+              class="settings btn"
+              type="button"
+              {{on "click" this.createGroupingPatternFromSelectedRows}}
+            >
+              <span>Create Grouping Pattern</span>
+            </button>
+
+            <button
+              class="settings btn danger"
+              type="button"
+              {{on "click" this.toggleGroupingPatternFromSelectedRows}}
+            >
+              <FaIcon @icon="times-circle" />
+              <span>Close</span>
+            </button>
+
+          {{else}}
+            <button
+              class="settings btn"
+              type="button"
+              {{on "click" this.toggleGroupingPatternFromSelectedRows}}
+            >
+              <FaIcon @icon="plus" />
+              <span>Grouping Pattern</span>
+            </button>
```

Comment:
These two could be the same button, a toggle, with the same icon and no label. Essentially a button that toggles the checkboxes on/off. Will take less space in the UI.